ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
> core.js must not list dependency on core.css or on any other CSS.

Yes

> The widgets will list its dependency on core.css themselves

No, Alex said dependency on `core.css` is implicit with any other css dependency. 
But I'm also okay with specifying it explcitly.

> There is no agreement on how button CSS should be defined yet.

I don't think this is a problem. Since this looks like it can land before the button rewrite, updating the CSS dependencies can happen in the button PR after this lands.